site/docs/main/customize-installation.md

```diff
@@ -95,10 +97,108 @@ velero client config set features=
 
 ## Customize resource requests and limits
 
-By default, the Velero deployment requests 500m CPU, 128Mi memory and sets a limit of 1000m CPU, 256Mi.
-Default requests and limits are not set for the restic pods as CPU/Memory usage can depend heavily on the size of volumes being backed up.
+At installation, Velero sets default resource requests and limits for the Velero pod and the restic pod, if you using the [restic integration](/docs/main/restic/).
 
-Customization of these resource requests and limits may be performed using the [velero install][6] CLI command.
+<table caption="Velero Customize resource requests and limits defaults" >
+  <tr><th>Setting</th><th>Velero pod defaults</th><th>restic pod defaults (Velero 1.4.2 and later)</th></tr>
+  <tr><td>CPU request</td><td>500m</td><td>500m</td></tr>
+  <tr><td>Memory requests</td><td>128Mi</td><td>512Mi</td></tr>
+  <tr><td>CPU limit</td><td>1000m (1 CPU)</td><td>1000m (1 CPU)</td></tr>
+  <tr><td>Memory limit</td><td>256Mi</td><td>1024Mi</td></tr>
+</table>
+
+### Install with custom resource requests and limits
+
+You can customize these resource requests and limit when you first install using the [velero install][6] CLI command.
+
+```
+velero install \
+  --velero-pod-cpu-request <CPU_REQUEST> \
+  --velero-pod-mem-request <MEMORY_REQUEST> \
+  --velero-pod-cpu-limit <CPU_LIMIT> \
+  --velero-pod-mem-limit <MEMORY_LIMIT> \
+  [--use-restic] \
+  [--default-volumes-to-restic] \
+  [--restic-pod-cpu-request <CPU_REQUEST>] \
+  [--restic-pod-mem-request <MEMORY_REQUEST>] \
+  [--restic-pod-cpu-limit <CPU_LIMIT>] \
+  [--restic-pod-mem-limit <MEMORY_LIMIT>]
+```
+
+### Update resource requests and limits after install
```

Comment:
Here's `kubectl patch` commands for the Velero `Deployment` and restic `Daemonset`:

`kubectl patch deployment velero -n velero --patch '{"spec":{"template":{"spec":{"containers":[{"name": "velero", "resources": {"limits":{"cpu": "4", "memory": "512Mi"}, "requests": {"cpu": "3", "memory": "256Mi"}}}]}}}}'`

`kubectl patch daemonset restic -n velero --patch '{"spec":{"template":{"spec":{"containers":[{"name": "restic", "resources": {"limits":{"cpu": "3", "memory": "4Gi"}, "requests": {"cpu": "1", "memory": "1Gi"}}}]}}}}'`

These values are samples. I tested them out against a running cluster and both applied cleanly.

Reply:
@nrb ok, i can update to use the patch commands instead of using the `kubectl edit`. These commands are long though, so most of them won't show to the reader without scrolling. Is there are way to break it up into different lines, like by using `/` at the end of the a line?

Reply:
@a-mccarthy Yeah, these can be broken with`\`, but _not_ within the actual JSON, the stuff inside the `'`, unfortunately. JSON's pretty picky about syntax, and that slash won't be valid.

I think it would look like this:

```
kubectl patch daemonset restic -n velero --patch \
'{"spec":{"template":{"spec":{"containers":[{"name": "restic", "resources": {"limits":{"cpu": "3", "memory": "4Gi"}, "requests": {"cpu": "1", "memory": "1Gi"}}}]}}}}'
```

Reply:
We can provide them as YAML files, but I think @ashish-amarnath's original goal was to provide a more compact command, though I may be wrong.